Re-order the model list in CI alphabetical order

cccclai opened the pull request:
Summary:
Re-order the model in the list in alphabetical order so it's easy to track

Rollback Plan:

Differential Revision: D79118824

digantdesai reviewed (approved):
🙏 

Leave a note saying keep this in order would keep it so :)

cccclai committed:
Re-order the model list in CI alphabetical order (#12929)

Summary:
Pull Request resolved: https://github.com/pytorch/executorch/pull/12929

Re-order the model in the list in alphabetical order so it's easy to track

Differential Revision: D79118824
